platform/core/src/measurements/classes/MeasurementApi.js

```diff
@@ -1,4 +1,5 @@
 import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
```

Comment:
Long-term, we don't want the `MeasurementApi` to depend on `cornerstone` or `cornerstone-core`, but that's outside the scope of this ticket. I'll create a follow-up issue for the backlog before we merge this PR.